frontend/src/pages/RecipientRecord/pages/CommunicationLogForm/pages/log.js

```diff
@@ -144,14 +144,14 @@ const Log = ({ datePickerKey }) => {
       </div>
       <div className="margin-top-2">
         <FormItem
-          label="Result "
+          label="Result"
           name="result"
+          required={false}
         >
           <Dropdown
-            required
             id="result"
             name="result"
-            inputRef={register({ required: 'Select a result of communication' })}
+            inputRef={register()}
```

Comment:
Result defaults to the below value:

![image](https://github.com/HHS/Head-Start-TTADP/assets/84350609/90723130-4dda-4d6a-8084-8448d03c33eb)

Do we want it to default to - Select - so they don't have any information they haven't set applied?

Reply:
Sure, I added a default disabled option

Reply:
Thanks, I think this makes more sense then saving something they didn't select.